### ISSUE TITLE
OpenGLCompute backend isn't being tested

### ISSUE THREAD

**Issue by steven-johnson** [halide/Halide#5627]
In Issue #5475 we decided to remove OpenGL but keep OpenGLCompute; AFAICT, we don't do any testing at all on the OpenGLCompute backend. The only thing in our code that appears to explicitly test it is `apps/openglcompute`, which is Android-specific (and hasn't been built in ~years). If we intend to keep this backend, we need to add some level of testing for it on the buildbots.

**Comment by shoaibkamil** [comment]
Agreed.  Would it be simplest to turn this on for Linux testing?  I realize those buildbots are pretty overtaxed, but I think testing for an additional backend isn't too bad (not like testing against an additional version of LLVM).

**Comment by steven-johnson** [comment]
Agreed. I'll enable that today and see how badly broken we are (as I assume we must be if we haven't been testing).

**Comment by steven-johnson** [comment]
Enabled it and I'm seeing LLVM errors of the form `"Duplicate option categories"` -- IIRC this is related to having multiple LLVM libraries in the open symbol space (i.e., one inside the driver, plus the one inside Halide)? 

(This is on a Linux box with Nvidia hw + drivers)

**Comment by shoaibkamil** [comment]
Yes, that does indeed seem like the same issue.  I worked around it in Makefile builds by setting the environment variable `WITH_LLVM_INSIDE_SHARED_LIBHALIDE=` before building, but of course this is NOT a long-term solution and requires carefully setting `LD_LIBRARY_PATH`s.

**Comment by steven-johnson** [comment]
See: https://github.com/halide/Halide/issues/4651
